web/scenes/Portal/Teams/TeamId/Team/Danger/page/index.tsx

```diff
@@ -29,24 +28,22 @@ export const TeamDangerPage = () => {
         <TeamProfile />
       </SizingWrapper>
 
-      <SizingWrapper gridClassName="order-2">
-        <div className="m-auto grid gap-y-8 py-8">
-          <div className="grid gap-y-3">
-            <Typography as="h1" variant={TYPOGRAPHY.H7}>
-              Danger zone
-            </Typography>
+      <SizingWrapper gridClassName="order-2 grow" className="flex flex-col">
+        <Section>
+          <Section.Header>
+            <Section.Header.Title>Danger zone</Section.Header.Title>
+          </Section.Header>
 
-            <p className="max-w-[36.25rem] text-grey-500">
+          <div className="grid justify-items-start gap-y-8 max-md:pb-8 md:max-w-[36.25rem]">
+            <p className="text-grey-500">
               This will immediately and permanently delete the team{" "}
-              <span className="font-medium text-gray-900">
-                {truncateString(fetchTeamQueryRes.data?.team_by_pk?.name, 30)}
-              </span>
+              <strong className="font-medium text-grey-900">
+                {fetchTeamQueryRes.data?.team_by_pk?.name}
```

Comment:
@michaelfurutin Try to name team something like `1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_1_` it will break mobile text. I think we should keep truncating.

Reply:
I'll check again, it should work without cutting using the function

Reply:
returned this functionality